css-ui-3/caret-color-016.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color dynamic changes</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
+<meta name="flags" content="dom">
+<meta name="assert" content="Test checks the style and copmuted style of caret-color property in different situations.">
```

Comment:
How about changing this to "checks that carret-color can be correctly changed using the style attribute, and that the computed value is done correctly."?